src/Service/OpenPlatform/SearchService.php

```diff
@@ -259,7 +259,7 @@ private function recursiveSearch(string $token, string $identifier, string $type
         switch ($type) {
             case IdentifierType::PID:
                 // If this is a search after a pid simply search for it and not in the search index.
-                $query = $identifier;
+                $query = 'rec.id='.$identifier;
```

Comment:
Is this related to reindex?

Reply:
Yes. It's a change that requires us to re-index